include/git2/describe.h

```diff
@@ -70,6 +69,16 @@ typedef struct git_describe_options {
 	GIT_DESCRIBE_DEFAULT_MAX_CANDIDATES_TAGS, \
 }
 
+/**
+ * Initialize git_describe_options structure
```

Comment:
This one and `git_describe_init_format_options` both miss the leading short doc string that you're consistently using for the other init_options functions now

Reply:
Fixed-up !